app/views/users/profile.html.erb

```diff
@@ -69,9 +69,14 @@
 
   <div>
     <i class="fas fa-user-friends"></i>
-    <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
-    &#8226;
-    <a href="/people/<%= @profile_user.username %>/following" class="link-secondary"><%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%></a>
+    <% if @profile_user.followers.count>0 %>
```

Comment:
This only checks if the number of followers is greater than 0 but doesn't check the following count. You need if-else statements for both `following` and `followers`. Something like this:
```
<% if @profile_user.followers.count > 0 %>
    <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
<% else %>
    <span class="text-secondary">0 followers</span>
<% end %>

```